src/create-broadcast.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Creates a broadcast that can be listened to, i.e. simple event emitter
+ *
+ * @see https://github.com/ReactTraining/react-broadcast
+ */
+
+const createBroadcast = initialValue => {
+  let listeners = [];
+  let currentValue = initialValue;
+
+  return {
+    publish(value) {
+      currentValue = value;
+      listeners.forEach(listener => listener(currentValue));
+    },
+    subscribe(listener) {
+      listeners.push(listener);
+
+      // Publish to this subscriber once immediately.
+      listener(currentValue);
+
+      return () => {
+        listeners = listeners.filter(item => item !== listener);
+      };
+    },
+  };
+};
+
+module.exports = createBroadcast;
```

Comment:
it does work, take a look:
* implementation https://github.com/iamstarkov/glamorous-ssr-theming/blob/master/pages/index.js#L4-L25
* deployed https://glamorous-ssr-theming.now.sh/  (disable javascript)

or execute this:

    $ curl https://glamorous-ssr-theming.now.sh/ | grep themed